pom.xml

```diff
@@ -48,6 +48,7 @@
         <module>ui</module>
         <module>forms</module>
         <module>persistence-mongodb</module>
+        <module>pmd-rules</module>
```

Comment:
we do not need to release versions of `pmd-rules` module. Use `maven-deploy-plugin` with `<configuration><skip>true</skip></configuration>`, to skip deploying it. we only need pmd rules during validate and check but we do not need to deploy `pmd-rules` as part of makueni artifact release lifecycle.